src/core/challenge_api.py

```diff
@@ -20,4 +24,37 @@ def vehicle_features_post(user_id: str):
 
     # TODO Implement the challenge
 
-    return make_response("OK", 200)
+    try:
+        data = request.json
+    except:
+        # fallback to parse request.data
+        try:
+            jsn = '{"vehicle"' + request.data.decode("utf-8").split('"vehicle"', 1)[1].split("--------------")[0]
+            data = json.loads(jsn)
```

Comment:
There probably is a better solution to this.
I think `request.get_json()` would solve the issue.

Reply:
Unfortunately not (win10, python 3.11, curl.exe/postman ), that's why I've made this awkward solution.